quarkus/service/src/test/java/org/apache/polaris/service/quarkus/catalog/IcebergCatalogTest.java

```diff
@@ -1693,8 +1695,8 @@ public void testFileIOWrapper() {
 
     table.updateProperties().set("foo", "bar").commit();
     Assertions.assertThat(measured.getInputBytes())
-        .as("A table was read and written")
-        .isGreaterThan(0);
+        .as("A table was read and written, but no trip to storage was made")
```

Comment:
nit: isn't the message supposed to describe failures as opposed to expected situation?

Reply:
Oh, that's true, the old message had me tripped up. Fixed, thanks!